apps/full-site-editing/full-site-editing-plugin/full-site-editing/plugins/close-button-override/index.js

```diff
@@ -43,10 +62,13 @@ domReady( () => {
 			'components-toolbar edit-post-fullscreen-mode-close__toolbar edit-post-fullscreen-mode-close__toolbar__override';
 		toolbar.prepend( componentsToolbar );
 
+		// These should go here so that they have any updates that happened while querying for the selector.
+		const { closeButtonLabel, closeButtonUrl } = fullSiteEditing;
+
 		// Create custom close button and append to components toolbar.
-		const newCloseButton = document.createElement( 'a' );
 		// When closing Template CPT (e.g. header) to navigate back to parent page.
```

Comment:
The two comment lines sound a bit awkward to me with the `newCloseButton` moved down. 🤔